editor/src/com/talosvfx/talos/editor/addons/scene/logic/GameObject.java

```diff
@@ -30,6 +30,8 @@ public class GameObject implements GameObjectContainer, Json.Serializable, IProp
     private String prefabLink = null;
 
     public boolean active = true;
+    private boolean locked = false;
+    private boolean visible = true;
```

Comment:
Maybe we call these more obvious names so that they arent confused with runtime properties.
For example editorLocked, or transformGizmoLocked. editorVisible etc.
THen we know its specifically a property only for the editor